### DISCUSSION TITLE
Dashboard Weekly Updates [26/08/24 - 30/08/24]

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#29004] [Changelog]
![screenshot-2024-08-30-at-13 18 28](https://github.com/user-attachments/assets/29aec90a-a524-40f3-827c-f469414aa6cd)

The SQL Editor got an upgrade this week, finally letting you organize snippets into folders!
- Favourites and Shared snippets are in folders now
- Organize Private snippets in folders as you like
- Share snippets with your team as you could before

Link: https://supabase.com/dashboard/project/_/sql/new
PR: https://github.com/supabase/supabase/pull/27881

### Other bug fixes and improvements
**Project compute size badge**
- See project compute details and upgrade right from the home screen ([PR](https://github.com/supabase/supabase/pull/23234))
 
**SQL Editor**
- Update the SQL Editor AI Assistant model to the latest from OpenAI ([PR](https://github.com/supabase/supabase/pull/28818))

**Comment by gc-ft** [comment]
Would it be possible to integrate the folders and other features for the snippets inside Supabase Studio? It would be great to have this for local development.

**Reply by saltcod** [reply]
Updates planned for local – hopefully soon. We do not save snippets at all in local at the moment. Very common request, hope to address soon.
